partTwo.asm

```diff
@@ -0,0 +1,150 @@
+###############################################
+
+	.text
+	
+		# The following lines load addresses for string printing, 
+		# specify a string is being printed and finally print the specified string.
+		
+		# This set of lines handles the printing of the introduction.
+		la   $a0, introOne	# Load address of the introOne string
+	      	li   $v0, 4		# Specify that you're printing a string
+	      	syscall			# Print the introOne string
+		
+		# newline
+		la   $a0, newLine       # load address of spacer for syscall
+		li   $v0, 4           # specify Print String service
+		syscall               # print the spacer string
+	      	
+	      	la   $a0, introTwo	# Load address of the introTwo string
+	      	li   $v0, 4		# Specify that you're printing a string
+	      	syscall			# Print the introTwo string
+		
+		# newline
+		la   $a0, newLine       # load address of spacer for syscall
+		li   $v0, 4           # specify Print String service
+		syscall               # print the spacer string
+	      	
+	      	la   $a0, introThree	# Load address of the introThree string
+	      	li   $v0, 4		# Specify that you're printing a string
+	      	syscall			# Print the introThree string
+		
+		# newline
+		la   $a0, newLine       # load address of spacer for syscall
+		li   $v0, 4           # specify Print String service
+		syscall               # print the spacer string
+	      	
+	      	# This set of lines handles the printing and storing of the user inputs.
+	      	# Printing statement to ask for input one:
+		la   $a0, entryOne	# Load address of the entryOne string
+	      	li   $v0, 4		# Specify that you're printing a string
+	      	syscall			# Print the entryOne string
+		
+		# Loading input one:
+		li $v0, 5		# Load the first number given.
+		syscall
+		
+		# Storing input one:
+		add $s0, $zero, $v0	# Store the first number given into $s0.
+		
+		# newline
+		la   $a0, newLine       # load address of spacer for syscall
+		li   $v0, 4           # specify Print String service
+		syscall               # print the spacer string
+		
+		# Printing statement to ask for input two:
+	      	la   $a0, entryTwo	# Load address of the entryTwo string
+	      	li   $v0, 4		# Specify that you're printing a string
+	      	syscall			# Print the entryTwo string
+	      	
+	      	# Loading input two:
+ 	    	li $v0, 5		# Load the second number given.
+		syscall
+		
+		# Storing input two:
+		add $s1, $zero, $v0	# Store the second number given into $s1.
+		
+		la $s2, calculations	# Load address of array into $s2.
+		
+		# Calculations performed in following lines:
+		# Adding inputs:
+		add $t0, $s0, $s1	# Adding inputs and storing in temp.
+		sw $t0, 0($s2)		# Storting sum into memory.
+		
+		# Subtracting inputs:
+		sub $t0, $s0, $s1	# Subtracting inputs and storing in temp.
+		sw $t0, 4($s2)		# Storing differnce into memory.
+		
+		# Multiplying inputs:
+		# Multiplication is tricky. I made input "A" add to itself for input "B" times.
```

Comment:
-2 Valiant effort in attempting to implement multiplication, and it works well for non-negative input, but crashes for negative input (negative B operand fails, negative A operand works) (Half credit for half a multiplication operation).  Among those who attempted this method, so far your's is the closest to being correct.  However, I recommend using 'mul' instead.